autoload/fzf/vim.vim

```diff
@@ -604,7 +604,18 @@ endfunction
 " ------------------------------------------------------------------
 
 function! s:get_git_root()
-  let root = split(system('git rev-parse --show-toplevel'), '\n')[0]
+  if get(g:, 'fzf_git_auto_chdir', v:true)
```

Comment:
Would use `1` for older versions of vim.
```suggestion
  if get(g:, 'fzf_git_auto_chdir', 1)
```

Reply:
Config removed as suggested